roles/docker-compose-service/README.md

```diff
@@ -0,0 +1,24 @@
+docker-service
+--------------
+
+This role configures docker-compose based services on each node
+
+The role is driven by a list of services defined in a list called `services` where each item is a dict which describes various service related configurations.
+
+
+The service list can be defined in the `hostvars` or `group_vars` ansible directory structure
+
+```yaml
+#sample service metadata
+
+service:
```

Comment:
should this be `services`?

If it's `service` it'll collide with ansible `service` resource

Reply:
let me confirm w/meta-data

Reply:
you are correct, it is services, I'll fix the readme